### ISSUE TITLE
TinyMce fails accessiblity test

### ISSUE THREAD

**Issue by mehtap0** [tinymce/tinymce#7346]
Hi,
 We currently use 8 years old version of Tinymce. 
Our web pages fail WCAG accessibility test , so I have decided to use your latest cloud version, but even that fails the test.

If I run "Accessiblity Insight for Web" on your own basic example page "https://www.tiny.cloud/docs/demo/basic-example/#"
It Fails with following rule error - 
aria-allowed-attr - Elements must only use allowed ARIA attributes.
LOCATION - 
div role="button" data-index="0" tab-index="-1" aria-level="1" tabindex="-1" class="tox-statusbar__path-item" aria-disabled="false">p

Can someone please advice if these errors can be ignored or there is some pluggin/init flag I need to set to fix this.

Regards
Punit
